queries/thrift/highlights.scm

```diff
@@ -110,6 +123,7 @@
 (boolean) @boolean
 
 (typedef_definition) @type.definition
+(namespace_definition) @type.definition
```

Comment:
What makes this different from namespace?

Reply:
thrift namespaces aren't normal namespaces per se, it's more like
`namespace lang foo`

so you can export a namespace to different languages with different names, e.g.

```thrift
namespace c_glib TTest
namespace cpp thrift.test
namespace delphi Thrift.Test
namespace go thrifttest
namespace java thrift.test
namespace js ThriftTest
namespace lua ThriftTest
namespace netstd ThriftTest
namespace perl ThriftTest
namespace php ThriftTest
namespace py ThriftTest
namespace py.twisted ThriftTest
namespace rb Thrift.Test
namespace st ThriftTest
namespace xsd test (uri = 'http://thrift.apache.org/ns/ThriftTest')
```